lib/normalizer.c

```diff
@@ -845,15 +845,9 @@ grn_nfkc_normalize_expand(grn_ctx *ctx,
                                     "");
 }
 
-/*
- * This function assumes that the input utf8_char is a valid UTF-8 character.
- * It is the caller's responsibility to ensure that utf8_char is valid UTF-8
- * before calling this function. This is a precondition in terms of contract
- * programming.
- */
-grn_inline static const unsigned char *
-grn_nfkc_normalize_unify_alphabet_diacritical_mark(
-  const unsigned char *utf8_char, unsigned char *unified)
+grn_inline static unsigned char *
+grn_nfkc_normalize_unify_a_diacritical_mark(const unsigned char *utf8_char,
```

Comment:
In general, common prefix is used for related functions in C:

```suggestion
grn_nfkc_normalize_unify_diacritical_mark_a(const unsigned char *utf8_char,
```

Reply:
fix: [ecb85d9](https://github.com/groonga/groonga/pull/1891/commits/ecb85d969529fb24a23aa69ed0a59cbe508ac343)